grunt/data/header-template-extended.txt

```diff
@@ -1,6 +1,6 @@
 /**
  * <%= package %> v<%= version %>
- *
+ * @license
```

Comment:
Let’s remove changes to this one file. Since we already have a header in the `React` module entry point, this causes license header to be outputted twice in the UMD build.